src/components/views/PasswordReset/NewPassword.js

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { message as alert } from 'antd';
+import 'antd/dist/antd.css';
+import { Link } from 'react-router-dom';
+import { applyReset } from '../../../redux/actions/passwordResetActions';
```

Comment:
Update import statements to use absolute paths

Reply:
ok